### ISSUE TITLE
sliderInput control inaccessible for keyboard or screen reader users

### ISSUE THREAD

**Issue by gtritchie** [rstudio/shiny#2845]
### System details

Output of `sessionInfo()`:

```
R version 3.6.3 (2020-02-29)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.4

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shiny_1.4.0.9002

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4.6         withr_2.1.2          packrat_0.5.0        digest_0.6.25        later_1.0.0         
 [6] mime_0.9             R6_2.4.1             jsonlite_1.6.1       xtable_1.8-4         magrittr_1.5        
[11] rlang_0.4.5          promises_1.1.0       tools_3.6.3          tinytex_0.22         rsconnect_0.8.16    
[16] httpuv_1.5.2         xfun_0.13            fastmap_1.0.1        compiler_3.6.3       htmltools_0.4.0.9003
```

### Example application *or* steps to reproduce the problem

Using the default app created by RStudio IDE 1.3.947, but specifically looking at:

```R
            sliderInput("bins",
                        "Number of bins:",
                        min = 1,
                        max = 50,
                        value = 30)
```

<img width="1131" alt="2020-04-24_14-58-13" src="https://user-images.githubusercontent.com/10569626/80260128-0b105180-863c-11ea-9e2f-73e85d1eb0f6.png">

### Describe the problem in detail
Load the page into browser, and try to use the slider via keyboard-only.

The slider control is completely inaccessible. You cannot get focus to it via keyboard (e.g. by Tab key), thus you cannot manipulate it via keyboard. Similarly, it is not annotated in a way that permits screen readers to recognize it as a slider, nor to manipulate it.

This violates numerous WCAG 2.1 level A requirements.

### Expected

- slider should be reachable via Tab sequence (i.e. tabindex=0)
- the slider should implement the aria 'slider' role both in markup and behavior:
https://www.w3.org/TR/wai-aria-practices/#slider

**Comment by jooyoungseo** [comment]
This issue has been being discussed in [ion.rangeSlider repo](https://github.com/IonDen/ion.rangeSlider/issues/708), and I will follow up with the developer.

**Comment by jooyoungseo** [comment]
From an accessibility standpoint, I think [noUiSlider](https://github.com/leongersen/noUiSlider) could be an alternative solution. However, it will take some time for us to tinker around for this binding.
